### ISSUE TITLE
Allow sane operation without config file

### ISSUE THREAD

**Issue by xtknight** [xtknight/mt7610u-linksys-ae6000-wifi-fixes#4]
In the cases where the config file is 0 bytes, there should still be a sane operation of the wireless adapter.

**Comment by mkreisl** [comment]
Yeah, as already written: setting the highest possible mode per default (15?) and adding some module parameters.
The rest will be done by wpa_supplicant
